start.py

```diff
@@ -1188,7 +1188,7 @@ def _configure_yarn(deployment, cluster, cluster_name):
                                                                configs)
 
 
-def _configure_for_streamsets_after_start(deployment, cluster_name, cluster, quiet):
+def _configure_for_streamsets_after_start(deployment, cluster_name, cluster, quiet, args):
```

Comment:
Same comment here as in other review about limiting the scope of args passed in (so `args` => `kerberos_principals`)